fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/MetadataManager.java

```diff
@@ -300,6 +303,65 @@ public long createTable(
                 "Fail to create table " + tablePath);
     }
 
+    public void alterTable(
+            TablePath tablePath, TableDescriptor tableDescriptor, boolean ignoreIfNotExists) {
+        // validate table properties before creating table
+        validateTableDescriptor(tableDescriptor, maxBucketNum);
```

Comment:
Currently we only support alter table properties. Should I fill the `Schema` here?